frontend/src/components/Login/Login.jsx

```diff
@@ -43,23 +48,31 @@ function Login(props) {
         .then((response) => {
```

Comment:
nit: I prefer await syntax.

Reply:
Sure, we can go for the await syntax. Do you think we should leave it like that in Login/Register but apply the await syntax in the rest of the code?